### ISSUE TITLE
Add the ability to track fibers between multiple frames

### ISSUE THREAD

**Issue by pantelisgeor** [ivan-usov/FiberApp#7]
At the moment, there is only the ability to fit and track fibers in single frames. The ability to track the same fibers between multiple frames (ie. in a video) would be ideal for a lot of people!

**Comment by ivan-usov** [comment]
Hey, I have a couple of questions.
- Do you want to keep the identity of fibrils between frames. In other words, is it necessary to know which fiber of the Nth frame corresponds to a fiber on the N+1 frame?
- What kind of analysis are you planning to perform with the tracked fibrils in the end? The answer to this question will also help to understand whether the first point is needed.

**Comment by pantelisgeor** [comment]
Ideally, yes. I don't know if you ever seen JFilament (ImageJ plugin), but that's the kind of tracking I was talking about. ie. Roughly trace the fiber in frame 1 and then let the program track it through the frames.

At the moment, I am tracking endoplasmic reticulum fibers and certain vesicles in human cells (two colour fluorescence microscopy videos). I am doind correlation analysis of the two throughout the frames, so ideally I would like to have the x-y coordinates of both in each frame. Hope that makes sense! If you need more information, please feel free to email me.

**Comment by ivan-usov** [comment]
Yes, I know JFilament. Actually, the cores of tracking algorithms in JFilament and FiberApp should be quite similar. I guess, you would get the same results/coordinates in both programs once you tried to compare them.
